stubs/prettytable/prettytable/prettytable.pyi

```diff
@@ -11,6 +12,9 @@ PLAIN_COLUMNS: int
 MARKDOWN: int
 ORGMODE: int
 RANDOM: int
+SINGLE_BORDER: int
+DOUBLE_BORDER: int
+BASE_ALIGN_VALUE: str
 
 class PrettyTable:
```

Comment:
This class appears to be missing the `header_align` property, and I'm not sure I understand why stubtest isn't picking up on it being missing: https://github.com/jazzband/prettytable/blob/4345954111637455549bb7dab1e6f3714c32b0a6/src/prettytable/prettytable.py#L663-L664

Reply:
It's not released https://github.com/jazzband/prettytable/blob/3.3.0/src/prettytable/prettytable.py

Reply:
Ah, that explains it!